### PR TITLE
Adds devices icon

### PR DESCRIPTION
**Pull request by badraxas**
Hello, in a project I needed an icon with a desktop device and a mobile device. So I did. I hope this icon can be useful to someone.
I removed the point in the smartphone because it is too big on this size.
image

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
Hi @badraxas,

Thanks for your contribution, I think this would actually be nice to have, but make sure to follow [our design guidelines](https://lucide.dev/docs/icon-design-guide), if I'm not mistaken, there isn't a 1px padding horizontally.

File: icons/devices.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg  xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M18 8V6c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2v7a2 2 0 0 0 2 2h8M10 19v-3.96 3.15M7 19h5"/><rect width="6" height="10" x="16" y="12" fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" rx="2" ry="2"/></svg>
```
Comment (karsa-mistmere) [inline icons/devices.svg:1]:
```suggestion
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M18 8V6a2 2 0 0 0-2-2H4a2 2 0 0 0-2 2v7a2 2 0 0 0 2 2h8" />
  <path d="M10 19v-3.96 3.15" />
  <path d="M7 19h5" />
  <rect x="16" y="12" width="6" height="10" rx="2" />
</svg>
```

**Review by karsa-mistmere — APPROVED** [review]

**Comment by ericfennis** [comment]
Shouldn't we call this `desktop-smartphone`, we had this name issue as well a bit the same with [`desktop-monitor`](https://lucide.dev/icon/monitor-speaker) what has a similar design as this one.

**Comment by badraxas** [comment]
According to [!217](https://github.com/lucide-icons/lucide/pull/217), I think they should be called `monitor-smartphone` to be consistent.

File: tags.json [hunk 2]
```diff
@@ -1278,6 +1278,15 @@
...
     "remove"
   ],
+  "devices": [
+    "smartphone",
+    "phone",
+    "cellphone",
+    "device",
+    "mobile",
+    "desktop",
+    "monitor"
```
Comment (karsa-mistmere) [inline tags.json:1288]:
```suggestion
    "monitor",
    ”responsive”
```

**Comment by karsa-mistmere** [comment]
> According to [!217](https://github.com/lucide-icons/lucide/pull/217), I think they should be named `monitor-smartphone` to be consistent.

Yeah, I second that and by the looks of it, @ericfennis also agrees, can you make the appropriate changes?

**Comment by badraxas** [comment]
Done :)

**Review by ericfennis — APPROVED** [review]
awesome!

**Pull request merged**